centaur/src/main/resources/standardTestCases/cwl_resources_papiv2.test

```diff
@@ -8,7 +8,7 @@ files {
 
 metadata {
   status: Succeeded
-  "outputs.machineTypeTool.machine_type": "projects/1005074806481/machineTypes/custom-2-6912"
+  "outputs.machineTypeTool.machine_type": "projects/1005074806481/machineTypes/custom-2-7168"
```

Comment:
[ToL] I hadn't seen this test before, it seems somewhat brittle if ever google decides to re-select their machine sizes? Oh well.... 🤷‍♂️